### ISSUE TITLE
Active nav color contrast is too low

### ISSUE THREAD

**Issue by EmmaDawsonDev** [StemWarriorHacks/StemWarriorHacks-Website#4]
The yellow text against blue/green background is very difficult to read because the contrast isn't sufficient. Suggest adding a slightly darker background behind the text or leaving the text black and adding a colored underline which highlights the active tab.
<img width="86" alt="Screenshot 2021-12-24 161932" src="https://user-images.githubusercontent.com/57045550/147362005-6914dfa0-1505-4b47-94c3-171650336b42.png">
